packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/models/dataservice/modelparts/ModelMetadata.java

```diff
@@ -15,6 +15,7 @@
 
 @Data
 @Accessors(chain = true)
+@Deprecated
```

Comment:
If we're marking this as deprecated would you mind throwing a javadoc with a note of what people should be using now?

Reply:
I think that came from me, ModelMetadata should be removed.